tasty/src/dotty/tools/tasty/besteffort/BestEffortTastyFormat.scala

```diff
@@ -0,0 +1,37 @@
+package dotty.tools.tasty.besteffort
+
+import dotty.tools.tasty.TastyFormat
+
+/*************************************************************************************
+Best Effort TASTy (.betasty) format extends the TASTy grammar with additional
+terminal symbols and productions. Grammar notation is kept from the regular TASTy.
+However, the lowercase prefixes describing the semantics (but otherwise not affecting
+the grammar) may not always hold.
+
+The following are the added terminal Symbols to the grammar:
+  * `ERRORtype` - representing an error from a previous compilation
+
+The following are the added productions to the grammar:
+
+  Standard-Section: "ASTs"
+```none
+  Type          = ERRORtype
+  Path          = ERRORtype
+```
+**************************************************************************************/
+object BestEffortTastyFormat {
```

Comment:
The `tasty/` directory contains the sources of the `tasty-core` artifact, which is a public library with some amount of compatibility guarantees.

We should not put this object in that artifact. It should be in the `compiler/` instead.